### PR TITLE
Allow navigating between all code libraries

### PR DESCRIPTION
**Pull request by kevmoo**
It's not angular, but it's easy.

Shows all dart libs at the top. You can click on each and see the diff.

[Review on Reviewable](https://reviewable.io/reviews/google/dart-shapeshift/33)

### CODE REVIEW

**Comment by kevmoo** [comment]
@srawlins ping? This actually allows you to navigate through all of the libraries

File: lib/src/frontend/js_zip_package_reporter.dart [hunk 1]
```diff
@@ -33,6 +35,19 @@ class JSZipPackageReporter extends PackageReporter {
         includeComments: includeComments));
   }
 
+  /// This can be *slow*, so it is async and it inserts a small delay before
+  /// processing each file to ensure UI – like a web page – can update.
```
Comment (srawlins) [inline lib/src/frontend/js_zip_package_reporter.dart]: :+1:

File: lib/src/frontend/js_zip_wrapper.dart [hunk 2]
```diff
@@ -103,17 +100,78 @@ void compareZips(VersionInfo leftVersion, ByteBuffer leftData,
...
+
+      _header.children.add(link);
+
+      _header.appendHtml(r' ');
+    }
+
+    _host.append(_header);
+
+    _content = new DivElement()
+      ..text = 'content here!'
```
Comment (srawlins) [inline lib/src/frontend/js_zip_wrapper.dart]: Should this be something else? Or removed?

**Comment by srawlins** [comment]
LGTM with one nit.

File: lib/src/frontend/js_zip_wrapper.dart (hunk 2 above)
Comment (kevmoo) [inline lib/src/frontend/js_zip_wrapper.dart]: Good call. Fixed.

**Pull request merged**